src/Apps/Order/Components/PhoneNumberForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { Flex, Spacer } from "@artsy/palette"
+import Input from "Components/Input"
+import React from "react"
+
+export type PhoneNumber = string
+
+export type PhoneNumberError = Partial<PhoneNumber>
+export type PhoneNumberTouched = boolean
+export type PhoneNumberChangeHandler = (phoneNumber: string) => void
+
+export const emptyPhoneNumber: string = ""
+
+export interface PhoneNumberFormProps {
+  onChange: PhoneNumberChangeHandler
+  value?: string
+  errors: PhoneNumberError
+  touched: PhoneNumberTouched
+  label: string
+}
+
+interface PhoneNumberFormState {
+  phoneNumber: string
+}
+
+export class PhoneNumberForm extends React.Component<
+  PhoneNumberFormProps,
+  PhoneNumberFormState
+> {
+  state = {
+    phoneNumber: this.props.value || emptyPhoneNumber,
+  }
+
+  changeEventHandler = phoneNumber => (
+    ev: React.FormEvent<HTMLInputElement>
+  ) => {
+    this.onChangeValue(phoneNumber, ev.currentTarget.value)
+  }
+
+  changeValueHandler = phoneNumber => (value: string) => {
+    this.onChangeValue(phoneNumber, value)
+  }
+
+  onChangeValue = (phoneNumber, value: string) => {
+    this.setState({ phoneNumber: value }, () => {
+      this.props.onChange(this.state.phoneNumber)
+    })
+  }
+
+  getError = (phoneNumber): string => {
+    return (this.props.touched && this.props.errors) || ""
+  }
+
+  render() {
+    return (
+      <Flex flexDirection="column">
+        <Input
```

Comment:
So the `Input` component here is wrapped in a Flex, but the only sibling that lives with it is a `Spacer`. 

It seems like we could just add an `mb` to `Input` and delete both the `Flex` and `Spacer`, right? Alternatively the parent component could add the spacer where necessary.

Reply:
Input doesn't take `mb` .but I can add it to the flex